hydra/README.md

```diff
@@ -0,0 +1,10 @@
+# Hydra Core Vocabulary and extensions
+
+URLs:
+- https://w3id.org/hydra/
+- https://w3id.org/hydra/extension/
+- https://w3id.org/hydra/extension/*/
+
+Contact: [Hydra Community Group](https://hydra-cg.com) <public-hydra@w3.org>
+
+W3ID created by Markus Lanthaler and Tomasz Pluskiewicz
```

Comment:
It would better to list the maintainers.  In this case I'm guessing it's the same, but better to be explicit.  Also would help to list github user ids.